client/state/data-layer/wpcom/sites/activity/test/index.js

```diff
@@ -80,13 +77,31 @@ describe( 'receiveActivityLog', () => {
 describe( 'receiveActivityLogError', () => {
 	it( 'should dispatch activity log error action', () => {
 		const dispatch = sinon.spy();
-		receiveActivityLogError( { dispatch }, { siteId: SITE_ID }, ERROR_RESPONSE );
-		expect( dispatch ).to.have.been.calledWith(
-			activityLogError( SITE_ID, {
-				error: 'unknown_blog',
-				message: 'Unknown blog',
-			} )
-		);
+		receiveActivityLogError( { dispatch } );
+		expect( dispatch ).to.have.been.called.once;
+		expect( dispatch.args[ 0 ][ 0 ] )
+			.to.be.an( 'object' )
+			.that.has.keys( [ 'type', 'notice' ] )
+			.that.includes( {
+				type: NOTICE_CREATE,
+			} );
+		expect( dispatch.args[ 0 ][ 0 ].notice )
+			.to.be.an( 'object' )
+			.that.has.keys( [
+				'noticeId',
+				'duration',
+				'showDismiss',
+				'isPersistent',
+				'displayOnNextPage',
+				'status',
+				'text',
+				'button',
+				'href',
+				'onClick',
+			] )
+			.that.includes( {
+				text: translate( 'Error receiving activity for site.' ),
+			} );
```

Comment:
I think this is a pretty heft integration test here that we probably don't need. What happens if the internals of the `createNotice()` API change? It will break this test which is supposedly examining the "failed to get activity" unit!

_if_ we really want to test the bejesus out of this thing then we should probably look at directly testing what we want: an error notice

```js
// default language of English shouldn't need `translate()` call but we could throw it in if we wanted
expect( dispatch ).to.have.beenCalledWith( errorNotice( 'Error receiving activity for site.' ) )
```